build.fsx

```diff
@@ -214,6 +214,42 @@ Target "GitHubRelease" (fun _ ->
 )
 
 // --------------------------------------------------------------------------------------
+// .NET CLI and .NET Core
+
+let isDotnetCLIInstalled = Shell.Exec("dotnet", "--info") = 0
```

Comment:
I think this line is crashing in AppVeyor CI

```
[00:01:46] Error: System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
[00:01:46] 	   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
[00:01:46] 	   at System.Diagnostics.Process.Start()
```

Note I think we could accept the PR if it is green (i.e. even if CoreCLR support is not running in CI - we can enable that next)